docs/contributing/governance/proposal-repository.md

```diff
@@ -0,0 +1,51 @@
+# Proposal Repository
+---
+Yearn Improvement Proposals (YIPs), Yearn Delegation Proposals (YDPs) and Yearn Signaling Proposals (YSPs) are all tools that token holders use to maintain and grow the protocol. 
+
+## Approved
+
+|Number|Title|Author|
+|------|-----|------|
+|[34](https://yips.yearn.finance/YIPS/yip-34)|Add Synthetix (SNX) to yVaults|[Substreight](https://github.com/substreight)|
+|[52](https://yips.yearn.finance/YIPS/yip-52)|Make Strategist Skin in Game Partner for Make Benefit of Glorious Brain of Yearn|[banteg](https://github.com/banteg), [lehnberg](https://github.com/lehnberg), [milkyklim](https://github.com/milkyklim)|
+
+## Implemented
+
+|Number|Title|Author|
+|------|-----|------|
+|[0](https://yips.yearn.finance/YIPS/yip-0)|YIP Purpose and Guidelines|Yearn Community|
+|[1](https://yips.yearn.finance/YIPS/yip-1)|Minting more YFI|[Andre Cronje](https://github.com/andrecronje)|
+|[10](https://yips.yearn.finance/YIPS/yip-10)|Transitionary YFI Only Voting|[Rewkang](https://github.com/rewkang)|
+|[12](https://yips.yearn.finance/YIPS/yip-12)|Reducing the quorum for accepting proposal|[cp287](https://github.com/illlefr4u)|
+|[32](https://yips.yearn.finance/YIPS/yip-32)|Remove YFI burning|[Sunil Srivatsa](https://github.com/alphastorm)|
+|[33](https://yips.yearn.finance/YIPS/yip-33)|Add LINK to yVaults|[franklin](https://github.com/franklin501)|
+|[35](https://yips.yearn.finance/YIPS/yip-35)|Distribute Donations vs Purchase YFI|[Andre Cronje](https://github.com/andrecronje),[Klim K](https://github.com/milkyklim)|
+|[36](https://yips.yearn.finance/YIPS/yip-36)|System rewards as operational capital|[Andre Cronje](https://github.com/andrecronje), iTo, [n00b](https://github.com/jchi18), [Artem K](https://github.com/banteg)|
+|[37](https://yips.yearn.finance/YIPS/yip-37)|Participate in CRV governance and 2.5x CRV reward boost|[Andre Cronje](https://github.com/andrecronje), [Artem K](https://github.com/banteg)|
+|[38](https://yips.yearn.finance/YIPS/yip-38)|Distribute / Keep Balancer Rewards|[Klim K](https://github.com/milkyklim)|
+|[39](https://yips.yearn.finance/YIPS/yip-39)|Add Curve sBTC Pool LP-Tokens yVault|[uhmpeps](https://github.com/az)|
+|[40](https://yips.yearn.finance/YIPS/yip-40)|Replace inactive multisig signers|[cp287](https://github.com/illlefr4u), [Artem K](https://github.com/banteg)|
+|[41](https://yips.yearn.finance/YIPS/yip-41)|Temporarily Empower Multisig|[tracheopteryx](https://github.com/tracheopteryx), [Joe Mahon](https://github.com/Substreight), [franklin501](https://github.com/franklin501), Michael Anderson, Vance Spencer|
+|[44](https://yips.yearn.finance/YIPS/yip-44)|Improve YIP categories|[sam bacha](sam@freighttrust.com)|
```

Comment:
You have to add `maito:` to sam@freighttrust.com otherwise it resolved as `yearn-devdocs/contributing/governance/sam@freighttrust.com`